### PR TITLE
RTG world by default + proxylicious proxy stuff.

### PR DESCRIPTION
**Pull request by whichonespink44**
This PR makes it so that the RTG world type is selected by default (configurable).

@srs-bsns There's also some proxy refactoring in here, so I won't merge this until the generator settings stuff has been merged  into dev.

### CODE REVIEW

File: src/main/java/rtg/event/GuiEventHandler.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ * @author @Adubbz (https://github.com/Adubbz)
+ *         <p>
+ *         Source: https://github.com/Glitchfiend/BiomesOPlenty/blob/962f1d0db5a8f04d601215ea79944254bcbd48f0/src/main/java/biomesoplenty/common/handler/GuiEventHandler.java
+ *         Modified by: WhichOnesPink (https://github.com/whichonespink44)
+ */
+public class GuiEventHandler {
+
+    public static GuiEventHandler instance = new GuiEventHandler();
+
+    @SideOnly(Side.CLIENT)
```
Comment (srs-bsns) [inline src/main/java/rtg/event/GuiEventHandler.java]: This annotation should go on the whole class since it's a client-only class.

Comment (whichonespink44) [reply src/main/java/rtg/event/GuiEventHandler.java]: Fixed in fba17c1

**Comment by Mako11** [comment]
Hello, I would like to say why the RTG mode - 1.10.2 is not working on more worlds, why just for the default superstition? I would also need other maps to use this generator maybe this adjustment? or possibly eliminating the blocking of other worlds with the same X-ray generator? I've seen the code

**Comment by Robijnvogel** [comment]
I don't think this is the correct place to contact the developers.
If you want to talk to them you could do that at the Minecraft Forums or [on Discord](https://discord.gg/0wIG7mz3g6TpkBsB).

**Pull request merged**